### DISCUSSION TITLE
Minimum Salary range for junior developers 💸

### DISCUSSION THREAD

**Discussion by kevinalfito69** [community/community#142533] [Programming Help]
### Body

This is very confusing for me because I see developer salaries in my country are very low even almost the same as other jobs, sometimes even lower, around 128 USD -384 USD per month for junior developers is that a normal salary in your place?

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by HosamShaban** [comment]
Unfortunately Yes,
I don't understand that.

**Comment by thomassiard** [comment]
If you’re looking to increase your earning potential, here are a few ideas that could help:

Explore Remote or Freelance Work: Many developers work for international clients remotely. Websites like Upwork, Toptal, or even open-source contributions can lead to paid opportunities that are globally competitive.

Upskill Continuously: Specialized skills in areas like data science, machine learning, cloud computing, or cybersecurity often come with higher demand and salaries. Certifications or open-source contributions can sometimes help you stand out.

Network with Other Developers Locally and Globally: Developer communities (on GitHub, LinkedIn, or Reddit) are great places to get insight into trends and opportunities that could increase your earning potential.

Look for Internships or Projects with Growing Companies: Sometimes startups or tech hubs offer competitive rates even in regions with lower salaries.

**Comment by baoanh177** [comment]
In Vietnam, the average salary for Junior FE dev ranges from 500 to 800 USD. If you are not satisfied with your salary, you can consider working as a freelancer for foreign projects. It's not a bad idea, but it's not easy either

**Comment by davlix** [comment]
maybe it depends on each country. if in my country someone with professional class is around $300. seems like we are one country :D

**Comment by jonyboy11** [comment]
In Italy, a junior developer’s gross salary might start around €24,000 per year, but after taxes and INPS contributions, the net is much lower. If you're curious, [calcolo-stipendionetto](https://calcolo-stipendionetto.it/) can show you exactly how much you'll take home based on region and contract type.
